service/controller/v11/resource/cloudformation/update.go

```diff
@@ -28,26 +30,26 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 
 		if stackStateToUpdate.ShouldUpdate && !stackStateToUpdate.ShouldScale {
 			// Fetch the etcd volume information.
-			etcdVolume := true
-			persistentVolume := false
-			volumes, err := r.ebs.ListVolumes(customObject, etcdVolume, persistentVolume)
+			filterFuncs := []func(t *ec2.Tag) bool{
+				ebs.NewDockerVolumeFilter(customObject),
+				ebs.NewEtcdVolumeFilter(customObject),
+			}
```

Comment:
This fixes updates with the new Docker volumes. We list them as well and detach them prior to the CF stack update.